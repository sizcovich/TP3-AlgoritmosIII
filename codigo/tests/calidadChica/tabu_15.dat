13 15 21 1725000
23 25 64 4661900
67 35 111 15701700
123 45 199 26685600
172 55 276 43643500
63 65 409 12522900
73 75 516 25070000
83 85 694 43232300
93 95 831 67146000
678 105 1054 152258300
