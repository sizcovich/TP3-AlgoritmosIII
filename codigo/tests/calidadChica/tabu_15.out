13 2 13 3 
23 2 2 18 
33 2 13 16 
123 11 1 35 12 18 22 30 34 38 41 43 44 
53 2 16 44 
258 16 21 6 10 12 13 17 23 24 29 31 32 35 42 47 51 60 
327 18 37 2 4 6 15 22 23 26 27 31 33 38 41 42 51 60 69 72 
443 21 1 60 6 7 10 16 23 25 27 28 34 45 48 51 52 53 62 63 67 75 85 
532 23 15 3 12 18 20 21 23 24 25 49 55 62 64 70 71 74 75 76 78 81 86 90 92 
103 2 61 15 
13 2 3 14 
38 6 5 4 13 23 24 25 
67 8 1 32 2 5 18 19 28 31 
13 2 3 8 
23 2 6 22 
33 2 30 18 
43 2 29 17 
53 2 1 41 
258 16 49 3 14 16 19 25 34 35 37 39 44 45 48 56 61 64 
73 2 59 29 
83 2 58 72 
93 2 1 73 
678 26 91 2 14 15 20 23 25 26 32 41 45 54 56 57 72 73 80 82 85 88 89 90 92 94 96 104 
